### PR TITLE
feat: Use -dev for development release instead of -next

### PR DESCRIPTION
**Pull request by mikolajb**
Let's switch to -dev for daily release.

### CODE REVIEW

File: make/repo.mk [hunk 1]
```diff
@@ -17,9 +17,10 @@ endif
 
 .PHONY: repo.dev.tag
 repo.dev.tag: ## Returns development tag
-repo.dev.tag:
-ifneq (,$(findstring -next,$(GIT_TAG)))
-	echo "$(GIT_TAG)"
+repo.dev.tag: install-tool.go.svu
+ifeq ($(GIT_CURRENT_BRANCH),main)
+	git fetch --tags
+	svu minor --pattern 'v[0-9].[0-9]{[0-9],}.[0-9]{[0-9],}' --suffix dev --tag-mode=all-branches --no-metadata
```
Comment (palexster) [inline make/repo.mk:23]: curiosity: what does --tag-mode=all-branches option introduce?

**Review by aweris — APPROVED** [review]

**Review by palexster — APPROVED** [review]
/lgtm

**Review by farhan5900 — APPROVED** [review]

**Review by rishabh96b — APPROVED** [review]

File: make/repo.mk (hunk 1 above)
Comment (jimmidyson) [reply make/repo.mk:23]: and in release branches, that should be removed and `svu minor` should become `svu patch`, right?

Comment (jimmidyson) [reply make/repo.mk:23]: As it does below :facepalm:

**Pull request merged**